Levi9.ERP.Domain/Services/ClientService.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Levi9.ERP.Domain.Models.DTO;
+using Levi9.ERP.Domain.Repositories;
+
+namespace Levi9.ERP.Domain.Services
+{
+    public class ClientService : IClientService
+    {
+        private readonly IClientRepository _clientRepository;
+        private readonly IAuthenticatationService _authenticationService;
+
+        public ClientService(IClientRepository _clientRepository, IAuthenticatationService _authenticationService)
+        {
+            this._clientRepository = _clientRepository;
```

Comment:
Parameters should not have an underscore prefix.  'this' is not necessary.
Look at the example for the ClientController.